tsup.config.js

```diff
@@ -0,0 +1,22 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+import { defineConfig } from 'tsup';
+
+export default defineConfig({
+  sourcemap: true,
+  clean: true,
+  minify: false,
+  entry: ['src'],
+  format: ['esm', 'cjs'],
+  outDir: 'dist',
+  target: 'node14',
```

Comment:
`package.json` references support for `node>=12` - are we intending on dropping that?

Reply:
We already have, good catch - I'll update where needed in my parking branch.